cloudfunctions/candidate_gmail_messages/cloudfunction.go

```diff
@@ -308,6 +323,57 @@ func (cf *CloudFunction) proccessMessages(messageIDs []string) error {
 			continue
 		}
 
+		parseRequest := ml.ParseJobRequest{
```

Comment:
Also assuming we only want to parse is the above filters don't fire

Reply:
Yes, and we want to go further where we only parse if the message is classified as a recruiting message below